app/usart_app/main.cc

```diff
@@ -0,0 +1,20 @@
+#include "board.h"
+
+uint8_t rxb[16] = "Message";
+std::span<uint8_t> tx_span(rxb, sizeof(rxb));
+
+uint8_t data[64];
+std::span<uint8_t> rx_span(data, sizeof(data));
```

Comment:
From what Nick and TJ told me, we should be using type safe template containers to store and send our data in C++. We can use std::array for this since we know the size of the message at compile time. We also don't need to convert the array into a std::span since the std::span parameter of your function also accepts C-Style arrays and std::arrays. We are only using the type std::span in the parameter because it allows us to pass in arrays of any size.
```suggestion
std::array<uint8_t, 16> tx_span = "Message";

std::array<uint8_t, 64> rx_span{};
```